platform-core-module/src/main/java/com/softicar/platform/core/module/uuid/UuidCache.java

```diff
@@ -0,0 +1,58 @@
+package com.softicar.platform.core.module.uuid;
+
+import com.softicar.platform.db.core.connection.DbConnectionOverrideScope;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+class UuidCache {
```

Comment:
We should have a test for this, since it's an extremely important core mechanism.

Reply:
implemented